html/global_attributes.json

```
@@ -1452,6 +1452,105 @@
           }
         }
       },
+      "nonce": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/nonce",
+          "spec_url": "https://html.spec.whatwg.org/multipage/urls-and-fetching.html#attr-nonce",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
```

```suggestion
              "version_added": "≤79"
```

EdgeHTML support might be complicated, see https://www.troyhunt.com/locking-down-your-website-scripts-with-csp-hashes-nonces-and-report-uri/